### ISSUE TITLE
Request regarding OMRSE terms in d-acts ('is-aggregate-of'; 'collection of organisms'; 'collection of humans'; and 'aggregate of organizations')

### ISSUE THREAD

**Issue by CDowland** [d-acts/d-acts#32]
OMRSE has recently added d-acts as an import. Currently, d-acts contains a small number of OMRSE classes (that were probably added via MIREOT), some of which are no longer in their current form, as well as an OMRSE object property that has been deprecated. As a result, these classes now appear in OMRSE with both the old and new annotations, where the old are being derived from the d-acts import. In one case, this leads to one of our terms having two parent classes, two labels, and two definitions. For this reason, we have a couple of requests:

1. We request the removal of the now deprecated OMRSE object property 'is-aggregate-of.'

2. We request the removal of your current versions of the following classes, and if needed, the replacement of them with their current versions:

- 'collection of organisms' (now labeled 'aggregate of objectual organisms') [http://purl.obolibrary.org/obo/OMRSE_00000022]
- 'collection of humans' [http://purl.obolibrary.org/obo/OMRSE_00000023]
- 'aggregate of organizations' [http://purl.obolibrary.org/obo/OMRSE_00000033]

**Comment by mbrochhausen** [comment]
Clint: would you want to make those changes? I am comfortable to give you write access to d-acts.

**Comment by mbrochhausen** [comment]
PS: Don't get me wrong, I can also take care of it, but it might be a good experience for you, too.

**Comment by CDowland** [comment]
Yes, I would be happy to do that.

**Comment by mbrochhausen** [comment]
Awesome! I'll give you permissions later today and give you an ok.

**Comment by CDowland** [comment]
I just noticed that some of the old annotations remained for 'aggregate of objectual organisms.' I have just corrected that in the development version and pushed the changes. 

Along with this, I made a correction to an annotation for 'collection of humans.' The definition was incorrectly annotated as an rdfs:comment. This was recently corrected in OMRSE, so I made the corresponding correction here.

I believe this task is now complete.
